ee/clickhouse/models/event.py

```diff
@@ -80,7 +81,7 @@ def create_event(
                     ),
                 )
             except:
-                pass
+                capture_exception()
```

Comment:
You'll need to do 
```
except Exception as err:
    capture_exception(err)
```

Reply:
This worked though: https://sentry.io/organizations/posthog/issues/2207757069/?project=1899813&query=is%3Aunassigned+is%3Aunresolved